src/functionBased/components/TodoItem.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { FaTrash } from 'react-icons/fa';
+import styles from './TodoItem.module.css';
+
+const TodoItem = ({
+  todo, handleChangeProps, deleteTodoProps, setUpdate,
+}) => {
+  const [editing, setEditing] = useState(false);
+
+  const handleEditing = () => {
+    setEditing(true);
+  };
+
+  const handleUpdatedDone = (event) => {
+    if (event.key === 'Enter') {
+      setEditing(false);
+    }
+  };
+
+  // useEffect(() => {
+  //   return () => {
+  //     console.log('Cleaning up...');
+  //   };
+  // }, []);
+
```

Comment:
I think it would be better to remove these lines of code for cleanness as they are not necessary.